chainerx_cc/chainerx/cuda/op_regist.h

```diff
@@ -6,3 +6,69 @@
 // Register an op statically in CudaBackend.
 #define CHAINERX_CUDA_REGISTER_OP(key_op_cls, op_cls) \
     static chainerx::internal::OpRegistrar<chainerx::cuda::CudaBackend, key_op_cls, op_cls> s_cuda_backend_op_##op_cls{};
+
+#define CHAINERX_CUDA_REGISTER_ELTWISE_DTYPE_UNARY_OP(func, func_def, visit_dtype)      \
```

Comment:
I think the following better describe the usage of the parameters. The "op_name" sort of depends on https://github.com/chainer/chainer/pull/6870/files#r275277089. 
```suggestion
#define CHAINERX_CUDA_REGISTER_ELTWISE_DTYPE_UNARY_OP(op_name, op_body, visit_dtype)      \
```

Reply:
True. Makes sense.

Reply:
Sorry I now noticed that we could align the first argument name with  `CHAINERX_{NATIVE,CUDA}_REGISTER_OP`, but we should do that in a separate PR.

Reply:
Addressed in https://github.com/chainer/chainer/pull/6941